javascript/javascript1/week2/homework/event-application.js

```diff
@@ -0,0 +1,12 @@
+
+function getEventWeekday(dayFromToday) {
```

Comment:
if dayFromToday is 1 then this function doesn't return the correct day. can you spot the issue?

Reply:
Thanks for the input. An extra % was missing :)